src/test/java/org/jenkinsci/plugins/envinject/EnvInjectBuildWrapperTest.java

```diff
@@ -1,27 +1,37 @@
 package org.jenkinsci.plugins.envinject;
 
-import static org.junit.Assert.assertEquals;
-import hudson.Util;
 import hudson.model.FreeStyleBuild;
 import hudson.model.FreeStyleProject;
-import hudson.model.Hudson;
-import hudson.model.Result;
-import junit.framework.Assert;
-
-import org.jenkinsci.lib.envinject.EnvInjectAction;
+import org.hamcrest.Matcher;
+import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.jvnet.hudson.test.CaptureEnvironmentBuilder;
 import org.jvnet.hudson.test.JenkinsRule;
 import org.jvnet.hudson.test.SingleFileSCM;
-import org.jvnet.hudson.test.CaptureEnvironmentBuilder;
 
-import java.io.IOException;
-import java.util.HashMap;
-import java.util.Map;
+import static com.google.common.collect.ImmutableMap.of;
+import static hudson.Util.replaceMacro;
+import static hudson.model.Result.SUCCESS;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasEntry;
+import static org.hamcrest.Matchers.hasKey;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.jenkinsci.plugins.envinject.matchers.WithEnvInjectActionMatchers.map;
+import static org.jenkinsci.plugins.envinject.matchers.WithEnvInjectActionMatchers.withEnvInjectAction;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assume.assumeThat;
 
 public class EnvInjectBuildWrapperTest {
 
-    public @Rule JenkinsRule j = new JenkinsRule();
+    @ClassRule
+    public static JenkinsRule j = new JenkinsRule();
```

Comment:
No, it can't - every test create new job which not intefere with each other. I've change it to class rule with attention to war between tests.